language/en/interface.json

```
@@ -571,6 +571,9 @@
 		"initialSpawn": {
 			"ready": "Ready",
 			"unready": "Unready",
+			"readied": "Readied",
+			"lock": "Press to Lock",
+			"unlock": "Press to Unlock",
```

```suggestion
			"lock": "Lock Position",
			"unlock": "Unlock Position",
```
To be clear what the lock/unlock is referring to

---

Makes sense to me